egs/hub4_english/s5/local/tuning/run_segmentation_wsj_b.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+set -e
+set -o pipefail
+
+# This script demonstrates how to re-segment long audios into short segments.
```

Comment:
This comment doesn't really explain what is going on here.  E.g are you using a WSJ model to segment shows of broadcast news for training purposes?  You can't exect people to work this out from the script, it's very complicated.
And  you need to comment out the WER results, this is a runnable scripts.